account_financial_report_qweb/static/src/css/report.css

```diff
@@ -96,11 +97,12 @@
 }
 
 .o_account_financial_reports_page {
-    background-color: @odoo-view-background-color;
+    background-color: white;
     color: @odoo-main-text-color;
     padding-top: 10px;
     width: 90%;
     margin-right: auto;
     margin-left: auto;
     font-family: Helvetica, Arial;
+    padding: 40px
```

Comment:
Why changing padding?

Reply:
so this looks better https://github.com/OCA/account-financial-reporting/pull/501#issuecomment-453060324

Reply:
Background color has nothing to do with padding.

Reply:
@i-vyshnevska pls post the difference w/ and w/out.
The issue is: it's really ugly w/out padding as you get the dark gray BG in touch w/ the borders and very next to the text.

Reply:
Take into account that this also affects PDF report, where a padding would affect to have 200 pages, or 400...

Reply:
@pedrobaeza  this padding not depend of amount of account entries if you have a lot of pages it add padding only in the end 
![image](https://user-images.githubusercontent.com/45663766/51234728-4373ab00-1976-11e9-8d20-3b75d05fda90.png)